platform/broadcom/sonic-platform-modules-ruijie/b6510-48vs8cq/sonic_platform/sfp.py

```diff
@@ -0,0 +1,1479 @@
+# -*- coding: utf-8 -*
+#!/usr/bin/env python
+
+#############################################################################
+# Ruijie B6510-48VS8CQ
```

Comment:
sonic has its sfp qsfp parse, why introduce your own parser?

Reply:
It's not introducing my own parser,  it is for new api adptation.